rotkehlchen/tests/db/test_db_upgrades.py

```diff
@@ -2025,6 +2025,18 @@ def test_upgrade_db_40_to_41(user_data_dir):
             assert names[0] in should_move_used_query_ranges | should_not_move_used_query_ranges
         cursor.execute('SELECT COUNT(*) FROM location WHERE location=? AND seq=?', ('m', 45))
         assert cursor.fetchone()[0] == 0
+        cursor.executemany(
+            'INSERT OR IGNORE INTO blockchain_accounts(blockchain, account, label) VALUES (?,?,?)',
+            [('eth', '0x2B888954421b424C5D3D9Ce9bB67c9bD47537d12', 'eth1_blockchain_accounts'),
+             ('eth', '0xc37b40ABdB939635068d3c5f13E7faF686F03B65', 'eth2_blockchain_accounts'),
+             ('btc', '0x2B888954421b424C5D3D9Ce9bB67c9bD47537d12', 'btc1_blockchain_accounts'),
+             ('btc', '0xc37b40ABdB939635068d3c5f13E7faF686F03B65', 'btc2_blockchain_accounts')],
+        )
+        cursor.executemany(  # eth1 and btc1 already exists in address_book
+            'INSERT OR IGNORE INTO address_book(address, blockchain, name) VALUES (?,?,?)',
+            [('0x2B888954421b424C5D3D9Ce9bB67c9bD47537d12', 'eth', 'eth1_address_book'),
+             ('0x2B888954421b424C5D3D9Ce9bB67c9bD47537d12', 'btc', 'btc1_address_book')],
+        )
```

Comment:
instead of adding them like this in the test add them in the database and check that you have what you are expecting